dust-tt/dust
# static file upload refactoring

## Description

- General refactoring -> split modals for table and document, use swr hooks, ...
- Removed all usage of pdfjs (and removed dep too)
## Risk

No major risk.
Can be rollbacked easily

## Deploy Plan
Simply deploy front

## Test
- Upload and update documents, tables, multiple
- Check that table picker is working properly
